CHANGES_NEXT_RELEASE

```diff
@@ -3,4 +3,5 @@
 - Fix: service header to use uppercase in case of update and delete (#1528)
 - Fix: Allow to send to CB batch update for multimeasures for NGSI-LD (#1623)
 - Add: new JEXL transformations for including into an array keys that have a certain value: valuePicker and valuePickerMulti
+- Add /iot/configGroups API endpoints (as equivalent to /iot/services) (#752)
```

Comment:
```suggestion
- Add /iot/configGroups API endpoints (as equivalent to /iot/services) (#752)
- Deprecated: /iot/services API routes
```

Reply:
Added here:https://github.com/telefonicaid/iotagent-node-lib/pull/1637/commits/e1f4e47cccf25b7ace81e33f9eb4801bfd60335c

This commit also includes an entry in deprecated.md file